packages/components/src/list/style.scss

```diff
@@ -3,15 +3,29 @@
 	padding: 0;
 }
 
+a.woocommerce-list__item {
```

Comment:
Since the default purpose is a link, this is some helper styling to stop the inner content from being made link color blue by default.